setup/install/deploy.md

```diff
@@ -39,20 +39,27 @@ deploy apply`.
 
 ## Connect to the Spinnaker UI
 
-If you have not enabled any sort of authentication, Spinnaker will not be
-publicly reachable by default. In this case, you will need to run the
-following command to reach the UI on [localhost:9000](http://localhost:9000):
+1. Run the following command:
 
-```bash
-hal deploy connect
-```
+   ```bash
+   hal deploy connect
+   ```
+
+   If necessary, set up an SSH tunnel to the host running Halyard.
```

Comment:
I might swap this line and the one below

Reply:
Yeah, I agonized over the order for a while, but I'd like to keep it like this. I feel like tunnelling just to run this command takes precedence over the (important but incidental) information that we're forwarding these ports.

Reply:
Ok, SG